File: Desafios Aleatorios/Java/exe05.java
import java.util.Scanner;

public class exe05 {
    public static void main(String[] args) {
        
        int n = 3;
        Scanner scanner = new Scanner(System.in);
        int somar = 0;

        for (int i = 0; i < n; i++) {
            System.out.print("Informe o " + (i+1) + "° tempo: ");
            int nota = scanner.nextInt();
            if (nota <= 0) {
                System.out.println("Valor inválido. Informe apenas números inteiros maiores que 0.");
                System.exit(1);
            }
            somar += nota;
        }

        scanner.close();

        double media = (double) somar / n;
        if (media <= 10) {
            System.out.println("CATEGORIA OURO");
        } else if (media <= 15) {
            System.out.println("CATEGORIA PRATA");
        } else if (media <= 20) {
            System.out.println("CATEGORIA BRONZE");
        } else {
            System.out.println("CATEGORIA DESCLASSIFICADO");
        }
    }
}

File: Desafios Aleatorios/Python/exe05.py

n = 3
somar = 0

for x in range(n):
    try:
        nota = int(input(f"Informe o {(1 + x)}° tempo: "))
    except ValueError:
        print("Erro: você deve digitar um número inteiro.")
        exit(1) 

    # valida a entrada fornecida
    if nota <= 0:
        print("Tempo inválido: o tempo deve ser maior que 0.")
        exit(1) 

    # soma a entrada do usuario
    somar += nota

# calcular a media 
media = float(somar) / n

# exibir o resultado
if media <= 10:
    print("CATEGORIA OURO")
elif media <= 15:
    print("CATEGORIA PRATA")
elif media <= 20:
    print("CATEGORIA BRONZE")
else:
    print("DESCLASSIFICADO")
